src/components/promoCode/services/promoCodeSrv.service.js

```diff
@@ -26,7 +33,7 @@
                 promoCodeStatusText[INVALID] = INVALID;
 
                 promoCodeSrv.checkPromoCode = function (promoCode, appContext) {
-                    var firebaseAppScopeName =  backendData[appContext].firebaseAppScopeName;
+                    var firebaseAppScopeName = backendData[appContext].firebaseAppScopeName;
```

Comment:
there are 2 places using promo code srv, and the use is different: 
1. in website - i need the all apps env and there i initializing the backendData[appContext].firebaseAppScopeName (in config phase). thats the reason i need the appContext map.
2. in app (student/dashboard) - i use the default backendData and i need only the appName (it is the student fb path).